### PR TITLE
Curiefense 2022 Annual Review

### PR DESCRIPTION
**Pull request by flaper87**

### CODE REVIEW

File: reviews/2022-curiefense-annual.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+- The project has been simplified and shrunken down to ease adoption and
+  deployment. This effort allowed the community to think through a good
+  deprecation process for major features and services.
+
+- The curiefense-helm code has been split into its own project, providing more
+  focus, clarity, and esae of navigation.
+
+- CI stabilization was a major focus for the team. During this period the team
+  focused on creating an e2e environment that would match a real produciton
```
Comment (sdake) [inline reviews/2022-curiefense-annual.md:52]: trivial - spelling error.

File: reviews/2022-curiefense-annual.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  environment at a smaller scale.
+
+## What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?
+
+
+**New Features / Refactoring**
+
+- The curiefense-helm project will be undergoing a major refactoring soon,
```
Comment (sdake) [inline reviews/2022-curiefense-annual.md:60]: line 48 is present tense - line 60 is past tense. I recognize the refactoring going on in curiefense-helm is new work. Unclear to me if new features are "roadmap items" or instead "new features "for this yearly review.

File: reviews/2022-curiefense-annual.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+**General improvements**
+
+- A major refactor to the test suite is in progress. The aim is to make the
+  tests agnostic to the environment, to make it simpler, and faster.
+
+**Documentation improvements**
+
+- Improve end user documentation. The team would like to simplify the quick
+  start guide so that it would be easier for newcomers to rampup and
+  familiarize themselves with the project.
```
Comment (sdake) [inline reviews/2022-curiefense-annual.md:74]: 👍

File: reviews/2022-curiefense-annual.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+
+## How can the CNCF help you achieve your upcoming goals?
+
+Curiefense has benefitted a lot from being part of the CNCF. The attention the
+project has gotten come not only from the hard work of the team but also
+because of the platform that the CNCF has provided.
+
+The same continus guidance and support is what the team would like to keep
```
Comment (sdake) [inline reviews/2022-curiefense-annual.md:83]: trivial spelling error.

File: reviews/2022-curiefense-annual.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+getting from the CNCF.
+
+There are currently no other requirements.
+
+## Do you think that your project meets the criteria for incubation?
+
+Curiefense is not ready to apply for incubation due to limited adoption and the
+lack of corporate diversity within its maintainers. The team is working on
+improving adoption, which will hopefully result in new maintainers and
+organizations joining the community. The team would like to work on this before
```
Comment (sdake) [inline reviews/2022-curiefense-annual.md:93]: "The team would like to improve adoption before".

**Comment by sdake** [comment]
LGTM @flaper87. Some minor comments about language. Hope you are well.

Cheers,
-steve

**Comment by dims** [comment]
Assigned to @TheFoxAtWork as the TOC shepherd for the 6/21 meeting.

**Comment by sdake** [comment]
Thanks @dims! @flaper87 I am a casual reviewer of these proposals, spotting any major problems, and focusing on work that is of personal interest. I don't actually approve PRs in this repository.

Thank you,
-steve

**Comment by TheFoxAtWork** [comment]
Recommendations:
* Improve the readme, balance between use of the project and community involvement information (meetings, slack, etc.) readme is a bit light on content
* Establish ongoing community meetings, make agendas, meeting notes, and other items for community management more accessible and easier to find, post meeting agenda and reminders in the slack channel to increase community involvement.
* Would recommend presenting at Security TAG by [filing a presentation issue](https://github.com/cncf/tag-security/issues/new?assignees=&labels=usecase-presentation%2C+triage-required&template=presentation.md&title=%5BPresentation%5D+Presentation+Title)

**Review by TheFoxAtWork — APPROVED** [review]
See recommendations comment on the PR, otherwise looks great!

**Pull request merged**